projects/client/RabbitMQ.Client/RabbitMQ.Client.csproj

```diff
@@ -38,8 +38,7 @@
   <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
     <DefineConstants>$(DefineConstants);CORECLR</DefineConstants>
   </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)' == 'SignedRelease' ">
-    <DelaySign>true</DelaySign>
+  <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
```

Comment:
I'd remove this `PropertyGroup` entirely. The only things in it that aren't redudant are `AssemblyOriginatorKeyFile` and `SignAssembly`, so you can move those up to the top `PropertyGroup` and get rid of everything else.